test/studies/shootout/k-nucleotide/benmcd/knucleotide-user-hash.chpl

```diff
@@ -0,0 +1,149 @@
+use IO, Map, Sort;
+
+config param tableSize = 2**16,
+             columns = 61;
+
+record intWrapper {
```

Comment:
Rather than `intWrapper`, what do you think about calling this something like `hashedInt` or maybe better, `hashVal` to reflect the fact that the values it stores are themselves hash values and not just any old integer?  To me, the name `intWrapper` begs the question "Why not just use an int?" whereas `hashedInt` or `hashVal` suggest a type that represents a pre-hashed value (which also helps explain why the `hash()` function just returns the value rather than doing any hashing of it).

Not related, but while talking about this record, I'd probably move it way down in the file since it isn't the first / most important thing to see.  Like maybe right before its first use in `writeCount()` or even down towards the bottom of the file where we have a bunch of helper code (e.g., near the `hash()` routine itself?)

Oh, which suggests an even better idea that combines these two:  Make the `hash()` routine return `intHash` or `hashVal` records directly rather than requiring the `new intWrapper()` calls in the code above.  This both reduces the cruft around the `new intWrapper(hash(...))` patterns that exist in multiple places (maybe just 2?) while also making the relationship between the `hash()` function (it computes the hash) and the `hashVal` record (it stores the value) much clearer.